api/apps/geoprocessing/src/marxan-sandboxed-runner/adapters-single/solutions-output/solutions-output.service.ts

```diff
@@ -13,6 +13,8 @@ import { SandboxRunnerOutputHandler } from '../../ports/sandbox-runner-output-ha
 @Injectable()
 export class SolutionsOutputService
   implements SandboxRunnerOutputHandler<ExecutionResult> {
+  #cancelled = false;
```

Comment:
AFAIR this service is injected once, so canceling one run would prevent all following runs to cancel the output dumping 🤔